### ISSUE TITLE
Cron expression Format not 6 Fields anymore according to godocs.

### ISSUE THREAD

**Issue by nevotheless** [containrrr/watchtower#398]
**Describe the bug**
Check title.

**Additional context**
According to the Watchtower docs: https://containrrr.github.io/watchtower/arguments/#scheduling A 6 Field cron format is getting used but the referenced godoc page https://godoc.org/github.com/robfig/cron#hdr-CRON_Expression_Format also only has 5 fields. 

I know that this was different like 1 or 2 weeks ago. Now's the question, is this only a change on the godocs page or is the watchtower cron implementation also affected by this?

**Comment by simonaronsson** [comment]
As we're locking our dependencies to specific commits using go modules, this change has yet to make it to watchtower. I'll go ahead and edit this issue to an improvement suggestion so that we keep it in mind next time we upgrade our dependencies. Thanks a bunch! 🙏🏼

**Comment by simonaronsson** [comment]
This is still an issue we need to address when updating. Reopening.

**Comment by totti4ever** [comment]
The referenced site now has 6 fields:
https://pkg.go.dev/github.com/robfig/cron@v1.2.0?tab=doc#hdr-CRON_Expression_Format

**Comment by simonaronsson** [comment]
> The referenced site now has 6 fields:
> https://pkg.go.dev/github.com/robfig/cron@v1.2.0?tab=doc#hdr-CRON_Expression_Format

Yes, but it's pointed at the version we're currently using. We should however update our dependencies, and when we do, this will change.
